### PR TITLE
Added section confirming we will maintain an in-person office

### PR DESCRIPTION
**Pull request by 0atman**
Some people find it better to work away from home, and some situations are better in-person. Added language to confirm we will still support this.

### CODE REVIEW

**Comment by 0atman** [comment]
Hope this language reads OK?

File: 001-work-from-home.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Eligible employees can choose to work from home up to three days per week, while
 
 The implementation of this policy will be reviewed and revisited regularly to ensure that it continues to benefit both the employees and the company.
 
+We will continue to maintian a much smaller office, for those employees and times where in-person, or outside-home works best.
```
Comment (InDieTasten) [inline 001-work-from-home.md:21]:
```suggestion
We will continue to maintain a much smaller office, for those employees and times where in-person, or outside-home works best.
```

**Review by InDieTasten — COMMENTED** [review]
Maybe explicitly mention the smaller office as solution for employees not eligible for WFH as well.

**Comment by 0atman** [comment]
test document upload

[test.docx](https://github.com/noboilerplate/DecisionRecord/files/13031353/test.docx)

**Comment by 0atman** [comment]
(just a demo)

**Pull request closed without merging**